wurst/objects/abilities/mage/hypnotist/StupefyField.wurst

```diff
@@ -13,7 +13,15 @@ import BuffObjEditing
 let AS_REDUCED = 3.0
 let MS_REDUCED = 3.0
 
-let TARGET_ALLOWED = "enemies,friend,ground,hero,organic,self,vulnerable"
+let TARGET_ALLOWED = commaList(
+    TargetsAllowed.enemies,
+    TargetsAllowed.friend,
+    TargetsAllowed.ground,
+    TargetsAllowed.organic,
+    TargetsAllowed.self,
+    TargetsAllowed.vulnerable,
+    TargetsAllowed.nonancient
```

Comment:
What's the point of adding non-ancient?

Reply:
Should this spell affect mammoth or not? I'd say no, just to avoid further abuses.

Reply:
OK, that's fine. I just wasn't sure that is what the effect was.